transformer/src/config/logger.py

```diff
@@ -20,6 +23,12 @@ def get_logger():
     if logger is not None:
         return logger
 
+    # When we are doing tests
+    if TEST:
```

Comment:
also, can't we factorize this block ?

Reply:
How would you do so?
Do you mean factorizing the `get_logger ` function across all the services?

Reply:
yes that's what I meant, kind of like the `analyzer` code

Reply:
Done